airflow/cli/cli_parser.py

```diff
@@ -671,7 +671,6 @@ def string_lower_type(val):
 ARG_UMASK = Arg(
     ("-u", "--umask"),
     help="Set the umask of celery worker in daemon mode",
```

Comment:
Help needs updating?

Reply:
Not currently -- it's only used for the `celery worker` command -- the other commands don't have a `--umask` argument, they only use the config value.